drivers/entropy/entropy_ateccx08.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2024 Vogl Electronic GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT microchip_ateccx08_entropy
+
+#include <errno.h>
+#include <string.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/entropy.h>
+#include <zephyr/drivers/eeprom/ateccx08.h>
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(entropy_ateccx08, CONFIG_ENTROPY_LOG_LEVEL);
+
+struct entropy_ateccx08_config {
+	const struct device *parent;
+};
+
+struct entropy_ateccx08_data {
+	uint8_t random_buffer[32];
```

Comment:
is this an arbitrary size ? 

Make it a define, it will be easier to interpret why this value.

Reply:
`atecc_random()` explicitly needs the 32 byte array, as the ATECC608 will always generate 32 bytes of randomness.